linux-tkg/linux-tkg-config/prepare

```diff
@@ -0,0 +1,1227 @@
+
+    if [ "${_aggressive_ondemand}" = "true" ]; then
+      msg2 "Applying MuQSS agressive ondemand governor patch"
+      patch -Np1 -i "$srcdir"/0004-glitched-ondemand-muqss.patch
+    fi
+
+    msg2 "Applying Glitched MuQSS patch"
+    patch -Np1 -i "$srcdir"/0004-glitched-muqss.patch
+
+  elif [ "${_cpusched}" = "upds" ] || [ "${_cpusched}" = "pds" ]; then
+    # PDS-mq
+    msg2 "Applying PDS base patch"
+    if [ "${_cpusched}" = "upds" ] || [ "$_basever" != "58" ]; then
+      patch -Np1 -i "$srcdir"/0005-v${_basekernel}_undead-pds099o.patch
+    else
+      echo "Project C!"
+      patch -Np1 -i "$srcdir"/0009-prjc_v${_basekernel}-r${rev}.patch
+    fi
+
+    # ondemand patches don't exist for 5.4
+    if [ "$_basever" = "58" ] && [ "${_cpusched}" = "upds" ];then
+      # is it dead or alive
+      doa="-undead"
+    fi
+    if [ "${_aggressive_ondemand}" = "true" ] && [ "$_basever" != "54" ] && [ "${_cpusched}" = "upds" ]; then
+      msg2 "Applying PDS agressive ondemand governor patch"
+      patch -Np1 -i "$srcdir"/0005${doa}-glitched-ondemand-pds.patch
+    fi
+
+    msg2 "Applying Glitched PDS patch"
+    patch -Np1 -i "$srcdir"/0005${doa}-glitched-pds_v${_basekernel}.patch
+
+  elif [ "${_cpusched}" = "bmq" ]; then
+    # Project C / BMQ
+    msg2 "Applying Project C / BMQ base patch"
+    if [ "$_basever" != "54" ]; then
+      patch -Np1 -i "$srcdir"/0009-prjc_v${_basekernel}-r${rev}.patch
+    else
+      patch -Np1 -i "$srcdir"/0009-bmq_v5.4-r2.patch
+    fi
+
+    if [ "${_aggressive_ondemand}" = "true" ] && [ "$_basever" != "54" ]; then
+      msg2 "Applying BMQ agressive ondemand governor patch"
+      patch -Np1 -i "$srcdir"/0009-glitched-ondemand-bmq.patch
+    fi
+
+    msg2 "Applying Glitched BMQ patch"
+    if [ "$_basever" != "54" ]; then
+      patch -Np1 -i "$srcdir"/0009-glitched-bmq_v5.7+.patch
+    else
+      patch -Np1 -i "$srcdir"/0009-glitched-bmq_v5.4.patch
+    fi
+
+  elif [ "${_cpusched}" = "cfs" ]; then
+    msg2 "Applying Glitched CFS patch"
+    patch -Np1 -i "$srcdir"/0003-glitched-cfs.patch
+  fi
+
+  if [ "$_distro" = "Void" ] && [[ "$_sub" = rc* ]]; then
+    cd ${wrksrc}/linux-${_rc_kern_ver}
+  elif [ "$_distro" = "Void" ] && [[ "$_sub" != rc* ]]; then
+    cd ${wrksrc}/linux-${_kern_ver}
+  fi
+
+  if [ "${_distro}" = "Arch" ] || [ "$_distro" = "Void" ]; then
+    if [ -z "${_configfile}" ]; then
+    _configfile="config_${_basekernel}.x86_64"
+    fi
+
+    cat "${srcdir}/${_configfile}" > ./.config
+  fi
+
+  # Set some -tkg defaults
+  echo "# CONFIG_DYNAMIC_FAULT is not set" >> ./.config
+  sed -i -e 's/CONFIG_DEFAULT_FQ_CODEL=y/# CONFIG_DEFAULT_FQ_CODEL is not set/' ./.config
+  echo "CONFIG_DEFAULT_CAKE=y" >> ./.config
+  echo "CONFIG_NR_TTY_DEVICES=63" >> ./.config
+  if [ "$_basever" = "54" ]; then
+    echo "CONFIG_TP_SMAPI=m" >> ./.config
+    echo "CONFIG_RAID6_USE_PREFER_GEN=y" >> ./.config
+    echo "# CONFIG_X86_P6_NOP is not set" >> ./.config
```

Comment:
This is also found on line 396 so it could be removed from here

Reply:
fixed